config/samples/crdb-tls-example.yaml.in

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 The Cockroach Authors
```

Comment:
Maybe this should be in a new directory `config/templates/samples` so that anything in `config/samples` can be copy pasted and run, wdyt?

Reply:
WFM. I thought that putting the templates next to the generated and checked in files would decrease the chances of us editing the generated files. I'd say, let's keep the current approach for a couple of releases and then move the files around.

Reply:
I think with a large enough warning that the files are generated it won't happen, maybe @taroface has some thoughts. I don't have a really strong opinion on this, we can keep and then re-evaluate later.

Reply:
Maybe we should add a CI check that would regenerate the files from templates and see if there is something not committed.

Reply:
I move the templates to `config/templates` and adde a CI check.